reference-implementation/lib/writable-stream.js

```diff
@@ -760,6 +782,7 @@ function WritableStreamDefaultControllerGetChunkSize(controller, chunk) {
   try {
     return strategySize(chunk);
   } catch (chunkSizeE) {
+    // TODO: Should we notify the sink of this error?
```

Comment:
I felt like this has been removed by you sometime before, but not fully sure and so I kept this while merging and forgot to mention this in the PTAL comments. Thanks for confirming. Removed.